packages/fx-core/src/core/middleware/projectMigrator.ts

```diff
@@ -524,6 +534,32 @@ async function getManifest(sourceManifestFile: string): Promise<TeamsAppManifest
   return await readJson(sourceManifestFile);
 }
 
+async function copyManifest(projectPath: string, fx: string, target: string) {
+  if (await fs.pathExists(path.join(projectPath, AppPackageFolderName))) {
+    await fs.copy(path.join(projectPath, AppPackageFolderName), target);
+  } else if (await fs.pathExists(path.join(fx, AppPackageFolderName))) {
+    // version <= 2.4.1
+    await fs.copy(path.join(fx, AppPackageFolderName), target);
+  } else {
+    // version <= 2.3.1
+    await fs.copy(path.join(fx, REMOTE_MANIFEST), path.join(target, REMOTE_MANIFEST));
+    const manifest: TeamsAppManifest = await getManifest(path.join(target, REMOTE_MANIFEST));
+    const color = (await fs.pathExists(path.join(fx, "color.png")))
```

Comment:
The icon may can be start with "https://"

Reply:
Added check.